boot/cmdline_test.go

```diff
@@ -209,5 +209,5 @@ func (s *kernelCommandLineSuite) TestComposeCommandLineManagedHappy(c *C) {
 	c.Assert(cmdline, Equals, "panic=-1 snapd_recovery_mode=recover snapd_recovery_system=20200314")
```

Comment:
this is also backwards now too, no?

Reply:
Yes, the price of splitting and sharing patches between PRs. I've untagled that in #9005, but I'll push a fix here and we can take care of any conflicts later.